main.ts

```diff
@@ -20,34 +26,84 @@ const DEFAULT_SETTINGS: PluginSettings = {
   lastUpdated: null,
   autoSync: true,
   highlightSortOldestToNewest: true,
+  syncNotesOnly: false,
   logFile: true,
   logFileName: 'Sync.md',
+  frontMatter: false,
+  frontMatterTemplate: `---
+id: {{ id }}
+updated: {{ updated }}
+title: {{ title }}
+author: {{ author }}
+---
+`,
+  headerTemplate: `
+%%
+ID: {{ id }}
+Updated: {{ updated }}
+%%
+
+![]( {{ cover_image_url }})
+
+# About
+Title: {{ title }}

```

Comment:
This was lost from the original version, please surround the sanitized title with `[[` `]]` so it supports linking by default:
![image](https://user-images.githubusercontent.com/4033621/149646216-75804459-6b9d-47b6-9755-19eea2649978.png)